components/Channel.php

```diff
@@ -81,6 +81,12 @@ public function defineProperties()
                 'type'        => 'dropdown',
                 'group'       => 'Links',
             ],
+            'includeStyles' => [
+                'title'       => 'rainlab.forum::lang.prop.include_css',
+                'description' => 'rainlab.forum::lang.prop.include_css_desc',
```

Comment:
language key should be `includeStyles` and `includeStyles_desc`

Reply:
without `prop`?

Reply:
Technically it would be better for it to be `rainlab.forum::lang.components.general.properties.includeCss`

Reply:
`rainlab.forum::lang.components.general.properties.includeCss`
or
`rainlab.forum::lang.components.general.properties.includeStyles`
?

Reply:
Language key corrected